src/commands/ext-dev-emulators-exec.ts

```diff
@@ -1,11 +1,12 @@
+import * as clc from "cli-color";
+
 import { checkMinRequiredVersion } from "../checkMinRequiredVersion";
 import { Command } from "../command";
-
+import { FirebaseError } from "../error";
 import * as commandUtils from "../emulator/commandUtils";
-import * as optionsHelper from "../extensions/emulator/optionsHelper";
 
 module.exports = new Command("ext:dev:emulators:exec <script>")
```

Comment:
Put a TODO in this file reminding us to remove this completely at the next major revision? Same in the other file.